### ISSUE TITLE
Set Optimization Strategy dropdown field according to Project Topping (in Wizard)

### ISSUE THREAD

**Issue by webrian** [opengisch/QgisModelBaker#874]
Als _kleine_ Unstimmigkeit im GUI ab Version 7.8.1 ist mir folgendes aufgefallen:
* Erzeuge ein QGIS Projekt von einer bestehenden Datenbank
* Auswahl der Datenbank (in meinem Fall ein GeoPackage, spielt aber vermutlich keine Rolle)
* im nächsten Schritt wähle ich das Topping aus unserem UsabilityHub aus
* das Dropdown-Feld «Project optimization strategy» wird deaktiviert, aber _immer_ mit dem Wert «Hide unused base class layers», auch wenn ich im Topping die «Group strategy» gesetzt habe
 
=>  ich denke, für eine etwas klarere Benutzerführung wäre es schön, wenn dieses Feld entsprechend dem Topping gesetzt würde und somit dem Benutzer auch angezeigt werden würde, welche Strategie das Topping vorgibt. Es ist richtig, dass es _nicht_ geändert werden kann.

![Bildschirmfoto vom 2024-01-08 11-13-55](https://github.com/opengisch/QgisModelBaker/assets/980073/c96a8fb9-8a90-4184-b28d-89fe27a7133d)

**Comment by signedav** [comment]
It's not super simple, since we parse the topping on "Generate" (after pressing "Erzeugen") - but still, maybe we can pre-parse or have an entry there called "topping-driven" or something.

**Comment by webrian** [comment]
Yes, I agree. Probably it is enough if the dropdown field says "topping-driven" or "from Project Topping" or something similar.
